### ISSUE TITLE
doesn't completely cut down larger trees

### ISSUE THREAD

**Issue by RyaanS** [gage-and-ryan-development/Lumberjack#4]

**Comment by morganlaneap** [comment]
When on `full` mode this happens a lot. Sometimes leaves get left and adjacent logs.

**Comment by RyaanS** [comment]
> When on `full` mode this happens a lot. Sometimes leaves get left and adjacent logs.

Correct. Unfortunately, I am no longer support this plugin as there is a wide selection of plugins that have better performance and algorithms than this one. Feel feel to make any pull requests you'd like and I'd review them if someone would like to still work on this plugin.
